docs/index.html.ejs

```diff
@@ -545,15 +545,26 @@
             </div>
           </div>
 
-          <br>
+          <br />
+
+          <div class="title-bar">
+            <div class="title-bar-text">A Maximised Title Bar</div>
```

Comment:
This might ought to be spelled "Maximized" with a Z for consistency with elsewhere in the project.

Reply:
My mistake. Duly updated.